charts/backend/templates/job-management-commands.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.managementCommands }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "signals-backend.fullname" . }}-management-commands
+  labels:
+    {{- include "signals-backend.labels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: api
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - sh
+            - "-c"
+            - |
+                /bin/bash <<'EOF'
```

Comment:
Instead of "EOF", maybe use a long random string, in case someone ever wants to use "EOF" in a management command.

Or, what would be wrong with the following?
```
command:
  - /bin/bash
  - "-c"
  - |
      {{ .Values.managementCommands | nindent 16 }}
```

Reply:
Thanks for your review 👍 just improved this part by removing "EOF".